src/welcome_bot.py

```diff
@@ -214,7 +212,7 @@ def more_info_requested(update: Update, context: CallbackContext) -> int:
         return ConversationHandler.END
     elif answer == "yes":
         return send_info_options(update, context)
-    elif InfoButtons.string_in_buttons(answer):
+    elif answer in INFO_BUTTONS:
         return info_requested(update, context)
```

Comment:
With the new button objects and the additional complexity created by events, it begins to feel like some of these handlers might move into classes and objects. Passing these variables is getting confusing to read.

I'm not sure how that would look like (a conversation singelton that that has associated buttons and responses?), but it's worth thinking about

Reply:
Maybe a conversation class that's eventually assigned a user and loads from a DB?